packages/core/src/property-editors/NumberEditor.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { PropertyEditorProps } from '../properties/property-editor-props';
+import { TextField } from '@material-ui/core';
+import { numberType } from '../properties/property-types';
+
+export interface TextEditorOptions {
+  maxLength?: number;
+  multiline?: boolean;
+  numberType: numberType;
```

Comment:
@nickpainter Maybe the enum should be a union string type so you can use it without importing enum first, and its usable with plain javascript aswell.

```typescript
numberType: "float" | "number" | "int"
```

Reply:
@alexbechmann  -  Good call. Updated to use union string type instead.